documentation/test/how_to_fail.md

```diff
@@ -0,0 +1,17 @@
+# All ways to fail a `test`
+There are three ways of failing a test, and marking it as having a validation error:
+* Having an `enforce` function which ends up failing.
+```js
+test('field', 'should fail by enforce', () => {
+    enforce(1).allOf({ largerThan: 5 });
+});
```

Comment:
because enforce throwing an exception... so in fact there is 2 ways of failing a test, I don't think you have to speak about enforce here

Reply:
If you want you can add a note to give an example with chai or enforce but there is not 3 ways